packages/browser/src/core/analytics/index.ts

```diff
@@ -34,9 +34,9 @@ const deprecationWarning =
   'This is being deprecated and will be not be available in future releases of Analytics JS'
 
 // reference any pre-existing "analytics" object so a user can restore the reference
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-const globalAny: any = global
-const _analytics = globalAny.analytics
+const _analytics: Analytics | undefined = (
+  (typeof window !== 'undefined' ? window : global) as any
```

Comment:
This looks safe to me since you're doing the typeof check on window.

I'm assuming `global` was used instead of `window` so that `AnalyticsNode` wouldn't break. This didn't break our UMD build since we use webpack and webpack was polyfilling `global` to use globalThis/this/window, and we expected other consumers to polyfill it.

I think everywhere else you removed `global` is never executed by `AnalyticsNode` so that's why we don't need the check in those places.

Reply:
Added a getGlobal function

https://github.com/segmentio/analytics-next/pull/502/files#diff-5e3d9ac53f16f5c22406dbdc9b21686a6b9694960dfac623c3340c14e19a6841